<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZScaleDialog</class>
 <widget class="QDialog" name="ZScaleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>181</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Z Scale Edition</string>
  </property>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QSlider" name="zscale_slider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>20</width>
     <height>270</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
   <property name="tickInterval">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="value_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset to 1.0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
